.gitignore

```diff
@@ -7,3 +7,4 @@ pkg/*
 .powder
 tmp
 bin
+.idea
```

Comment:
Please can you remove this change? Not project related ignored files can be part of local global gitignore. See https://help.github.com/articles/ignoring-files/#create-a-global-gitignore for more info.